vscode-dotnet-runtime-library/install scripts/interprocess-communicator.sh

```diff
@@ -0,0 +1,36 @@
+#!/usr/bin/env bash
+EXECFOLDER=$1 # First argument is the working folder as this is launched with cwd of /root
+OKSIGNALFILE="$EXECFOLDER/ok.txt"
+COMMANDTORUNFILE="$EXECFOLDER/command.txt"
+OUTPUTFILE="/home/viru/om.txt"
+end=$((SECONDS+3600))
+while true
+do
+        stop=false
+        while [ $SECONDS -lt $end ];
+        do
+            if test -f "$COMMANDTORUNFILE"; then
+                # echo "COMMAND FILE FOUND" >> "$OUTPUTFILE" # Leave this here as an example of debugging
+                COMMAND="$(cat "$COMMANDTORUNFILE" | awk '{$1=$1;print}')"
+                for validCmd in "${@:2}"
+                do
+                    if [ "$COMMAND" == "$validCmd" ]; then
+                        IFS=' ' read -ra COMMANDARGS <<< "$COMMAND"
```

Comment:
Eventually we should split the cmd file to be line by line instead of space separated, but it works for now because of the commands were running under sudo

Reply:
```suggestion
                        # Eventually we should split the cmd file to be line by line instead of space separated,
                        # but it works for now because the commands are running under sudo
                        IFS=' ' read -ra COMMANDARGS <<< "$COMMAND"
```